### PR TITLE
f-account-info@v0.20.0 - Added success & failed alerts to the Save button

### PR DESCRIPTION
**Pull request by oliversweb**
### Added
- Alerts for failed and successful saving.

### CODE REVIEW

File: packages/components/pages/f-account-info/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "fozzie"
   ],
   "scripts": {
-    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
```
Comment (zlatinivanov-je) [inline packages/components/pages/f-account-info/package.json]: Mystery white space.

File: packages/components/pages/f-account-info/src/components/Notifications.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<template>
+    <div>
+        <f-alert
+            v-if="showSaveErrorAlert"
```
Comment (zlatinivanov-je) [inline packages/components/pages/f-account-info/src/components/Notifications.vue:4]: Is it possible for both alerts to appear at the same time?

**Review by zlatinivanov-je — COMMENTED** [review]
A couple of small questions, other than that LGTM

File: packages/components/pages/f-account-info/src/components/Notifications.vue (hunk 2 above)
Comment (oliversweb) [reply packages/components/pages/f-account-info/src/components/Notifications.vue:4]: No

**Review by zlatinivanov-je — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**